src/backend/vulkan/src/lib.rs

```diff
@@ -303,8 +288,16 @@ unsafe extern "system" fn debug_report_callback(
 
 impl hal::Instance<Backend> for Instance {
     fn create(name: &str, version: u32) -> Result<Self, hal::UnsupportedBackend> {
-        // TODO: return errors instead of panic
-        let entry = VK_ENTRY.as_ref().map_err(|e| {
+        #[cfg(not(feature = "use-rtld-next"))]
+        let vk_entry = Entry::new();
```

Comment:
could we just do a `match Entry::new()` here instead of following with `map_err` later?